test/show.jl

```diff
@@ -1014,4 +1014,132 @@ end
            1 │ 1"""
 end
 
+@testset "InlineStrings with GroupedDataFrame" begin
+    df = DataFrame(id=inlinestrings(["a", "b", "c"]), value=collect(Int64, 1:3))
+
+    io = IOContext(IOBuffer(), :limit=>true)
+    show(io, groupby(df, :id))
+    @test String(take!(io.io)) === "GroupedDataFrame with 3 groups based on key: id\n" *
```

Comment:
Why not use a triple-quoted string instead?

Reply:
I was about to write the same comment! Something like that:

```julia
"""
GroupedDataFrame with 3 groups based on key: id
First Group (1 row): id = "a"
 Row │ id       value
     │ String1  Int64
─────┼────────────────
   1 │ a            1
⋮
Last Group (1 row): id = "c"
 Row │ id       value
     │ String1  Int64
─────┼────────────────
   1 │ c            3
"""
```

Reply:
I will re-check it (this is what I did initially, but I was getting test errors constantly for some reason I did not understand)

Reply:
OK - I have managed to find problems. The issues were trailing whitespace in some cases.